ldap/servers/slapd/back-ldbm/db-mdb/mdb_import_threads.c

```diff
@@ -785,15 +829,20 @@ dbmdb_import_entry_info_by_param(EntryInfoParam_t *param, WorkerQueueData_t *wqe
             } else {
                 param->ekey.mv_data = (void*)slapi_sdn_get_ndn(&param->sdn);
                 param->ekey.mv_size = strlen(param->ekey.mv_data)+1;
-                param->pkey.mv_data = (void*)slapi_dn_find_parent_ext(param->ekey.mv_data, 0);
+                param->pkey.mv_data = (void*)slapi_dn_find_parent_ext(param->ekey.mv_data, dnrc != DNRC_OK);
                 param->pkey.mv_size = strlen(param->pkey.mv_data)+1;
                 if (!param->pkey.mv_data) {
                     dnrc = DNRC_NOPARENT_DN;
                 }
             }
         }
     }
-    if (dnrc == DNRC_OK) {
+    if (param->eid == 1 && dnrc == DNRC_RUV) {
+        /* RUV is first entry lets return it as it */
```

Comment:
Does it assume that the RUV is the first entry in the ldiffile ? What if it is not ?

Reply:
Indeed this test goal is to handle the" RUV entry is before suffix entry" case 
 if the RUV is not the first entry, the suffix entry is supposed to be the first entry
    (otherwise the ldif is buggy and all entries before the ruv get skipped) .
 and in this case the RUV is handled as standard entry 
     ( looking for the parent (i.e the suffix) entry info and adding the entry info)

Reply:
Thanks